frontend/manage_users.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file This file contains the Manage function. This function is used for
+ * granting and revoking permissions from users, as well as approving requests
+ * for adding individuals to herds in the genebanks you manage.
+ */
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import { Tabs, Tab } from '@material-ui/core/'
+import { Box, Button } from '@material-ui/core'
+
+import { useDataContext } from './data_context'
+import { ManageUser } from '~manage_user';
+
+// Define styles for tab menu
+const useStyles = makeStyles({
+  sidebar: {
+    float: "left",
```

Comment:
Why not put sidebar and controls side-by-side using flex instead of float?

Reply:
Then you don't need to do all the calc.

Reply:
I honestly don't remember. I almost always use flex, so I want to think I had a reason for a float, but I can't remember what it would be :D I'll have a look through the rules once we have some more data on what they want from the thing though! I'll probably design everything like a big flex-thing in the end...

Reply:
Alright!